packages/cactus-plugin-satp-hermes/src/main/typescript/cross-chain-mechanisms/oracle/implementations/oracle-evm.ts

```diff
@@ -105,7 +103,7 @@ export class OracleEVM extends OracleAbstract {
     };
 
     this.id = this.options.leafId || this.createId(EthereumLeaf.CLASS_NAME);
-    this.keyPair = options.keyPair || Secp256k1Keys.generateKeyPairsBuffer();
+    this.keyPair = Secp256k1Keys.generateKeyPairsBuffer();
```

Comment:
the user should be able to provide a keypair (of already initialized keys) to the oracle 
```suggestion
    this.keyPair = options.keyPair || Secp256k1Keys.generateKeyPairsBuffer();
```

Reply:
No. I initially misunderstood the required refactor and changed the keyPair config parameter and associated logic instead of the pubKey parameter. When reverting the changes, I missed this subtlety. Thanks for pointing it out.